src/object.d

```diff
@@ -1964,15 +1964,15 @@ V[K] dup(T : V[K], K, V)(T* aa)
     return (*aa).dup;
 }
 
-auto byKey(T : V[K], K, V)(T aa) pure nothrow @nogc
+auto byKey(T : V[K], K, V)(T aa) pure nothrow @nogc @trusted
 {
     import core.internal.traits : substInout;
 
     static struct Result
     {
         AARange r;
 
-    pure nothrow @nogc:
+    pure nothrow @nogc @trusted:
```

Comment:
Seeing `@trusted` followed by `:` makes me nervous... IMO it's better to add it to each function signature individually.